delfin/coordination.py

```diff
@@ -112,6 +119,79 @@ def get_lock(self, name):
 LOCK_COORDINATOR = Coordinator(prefix='delfin-')
 
 
+class LeaderElectionCoordinator(Coordinator):
+
+    def __init__(self, agent_id=None):
+        super(LeaderElectionCoordinator, self). \
+            __init__(agent_id=agent_id, prefix="leader_election")
+        self.group = None
+
+    def start(self):
+        """Connect to coordination back end."""
```

Comment:
Can we make it switchable coordination back end based on config, current implimentaion can be only for redis with toox

Reply:
created an issue to address the issue as future improvement https://github.com/sodafoundation/delfin/issues/610